scripts/stageapi/stage/roomgfx.lua

```diff
@@ -142,6 +145,12 @@ function StageAPI.LoadBackdropSprite(sprite, backdrop, mode) -- modes are 1 (wal
 
     sprite:Play(shapeName, true)
 
+    if mode ~= 2 then
+        for layer, chosenWall in pairs(usedData) do
```

Comment:
Is there any particular point to running the callback for every layer individually rather than just passing in usedData directly and running it once?

Reply:
Updated